layers/+lang/latex/funcs.el

```diff
@@ -20,6 +20,14 @@
     ;; (when build-proc
     ;;   (set-process-sentinel build-proc 'latex//build-sentinel))))
 
+(defun latex/all ()
+  "Save, build and view in one command."
+  (interactive)
+  (progn
+    (let ((TeX-save-query nil))
+      (TeX-save-document (TeX-master-file)))
```

Comment:
@TheBB Thanks for the comment :smiley: . 

In my case, it is a little annoying that the `TeX-command-run-all` keeps asking me to save the .tex files when I decide to compile them. I just need my files to be compiled after I finish editing them without asking me to do the saving. So I would assume that the users know their files would be saved before compiling, otherwise they would not compile the files.